src/App.tsx

```diff
@@ -109,6 +109,21 @@ export default class App extends Component<{}, { chosenNode: NodeID, value: stri
 
     }
 
+    handleResize()
+    {
+        this.setState(prevState => prevState);
+    }
+
+    componentDidMount()
+    {
+        window.addEventListener("resize", this.handleResize.bind(this));
+    }
+
+    componentWillUnmount()
+    {
+        window.removeEventListener("resize", this.handleResize.bind(this));
```

Comment:
Will the removal work? Seams like they're two distinct function objects.

Reply:
IDK, but they told me so on the internet.

Reply:
It's a very good style btw